website/content/en/preview/getting-started/_index.md

```diff
@@ -87,23 +87,19 @@ Karpenter requires permissions like launching instances. This will create an AWS
 
 This step is only necessary if this is the first time you're using EC2 Spot in this account. More details are available [here](https://docs.aws.amazon.com/batch/latest/userguide/spot_fleet_IAM_role.html).
 
-```bash
-aws iam create-service-linked-role --aws-service-name spot.amazonaws.com
-# If the role has already been successfully created, you will see:
-# An error occurred (InvalidInput) when calling the CreateServiceLinkedRole operation: Service role name AWSServiceRoleForEC2Spot has been taken in this account, please try a different suffix.
-```
+{{% script file="./content/en/preview/getting-started/scripts/step06-add-spot-role.sh" language="bash"%}}
```

Comment:
These aren't formatting in the enriched file. Is this Hugo syntax and not markdown syntax?

Reply:
What do you mean by "These aren't formatting in the enriched file. " ?

Reply:
Oops, meant to say rendered, but realized this is Hugo's syntax.